addons/website/views/snippets/s_cta_card.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_cta_card" name="Card Call to Action">
+    <section class="s_cta_card o_cc o_cc2 pt64 pb64" data-oe-shape-data="{'shape':'web_editor/Blocks/03'}">
+        <div class="o_we_shape o_web_editor_Blocks_03"/>
+        <div class="container">
+            <div class="row align-items-center">
+                <div class="col-lg-9">
+                    <h2>50,000+ companies run Odoo to grow their businesses.</h2>
+                    <p class="lead">Join us and make your company a better place.</p>
+                </div>
+                <div class="col-lg-3">
+                    <div class="s_card card o_cc o_cc1 ms-auto shadow" data-name="Card" style="border-width: 0;" data-vcss="001">
+                        <div class="card-body p-4">
+                            <h5 class="card-title">What you will get</h5>
+                            <br/>
+                            <p><i class="fa fa-check rounded-pill text-o-color-1" role="img"/>&#160;&#160;Wonderful experience</p>
```

Comment:
`rounded-pill`: remove these classes because will invalidate user edition.

| |
|--------|
| ![image](https://github.com/user-attachments/assets/1e00f640-e2af-4460-b49c-8f73b9421596) |